conf/app.routes

```diff
@@ -265,9 +265,12 @@ GET        /update-record/:recordId/withdraw-request/success
 
 GET        /update-record/:recordId/withdraw-request/reason                                 controllers.ReasonForWithdrawAdviceController.onPageLoad(recordId: String)
 POST       /update-record/:recordId/withdraw-request/reason                                 controllers.ReasonForWithdrawAdviceController.onSubmit(recordId: String)
+GET        /problem/expired-commodity-code                                                  controllers.ExpiredCommodityCodeController.onPageLoad(recordId:String)
```

Comment:
i think you need to have a record id in this url because otherwise if i go straight to that url (type it into the address bar) it says bad request because the record id will be null

Reply:
I believe this should be navigated through previous page, url hacking isn't fair. Also, this url came from design, and recordId actually isn't needed.

Reply:
i think we should push back and get the recordId in the url - i'm sure if you explain to design they would be happy for the recordId to be in it

Reply:
because if not we have urls that are broken, which just feels like the wrong way to build an app

Reply:
yeah, sorry didn't realise about potential issues - it's updated now please.